docs/serving/samples/knative-routing-go/README.md

```diff
@@ -174,8 +191,26 @@ You should see: `Login Service is called !`
 kubectl apply --filename docs/serving/samples/knative-routing-go/routing.yaml
 ```
 
+If you have already had an existing domain name for your service, please replace all
+the presences of "example.com" in routing.yaml with "<your domain name>" for
+spec.hosts and spec.http.rewrite.authority.
+
+In addition, you need to verify how your domain template is defined. By default, we
+use the format of {{.Name}}.{{.Namespace}}, like search-service.default and
+login-service.default. However, some Knative environments may use other format like
+{{.Name}}-{{.Namespace}}. You can find out the format by running the command:
+
+```
+kubectl get cm  -n knative-serving config-network -o yaml
+```
+
+then look for the value for `domainTemplate`. If it is `{{.Name}}-{{.Namespace}}.{{.Domain}}`,
+you need to change `search-service.default` into `search-service-default` and
+`login-service.default` into `login-service-default` as well in routing.yaml.
```

Comment:
I would use back ticks for `routing.yaml`

Reply:
Done. thx.